backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -88,7 +129,7 @@ public SqlDatabase(DataSource source, boolean init) {
   }
 
   @Override
-  public void init() { // setup default stuff
+  public synchronized void init() { // setup default stuff
```

Comment:
if the method is synchronized, why is the dup key error being catched and hidden ?

Reply:
ah, that is not necessary anymore